superset-frontend/src/components/WarningIconWithTooltip/index.tsx

```diff
@@ -24,11 +24,13 @@ import { Tooltip } from 'src/components/Tooltip';
 export interface WarningIconWithTooltipProps {
   warningMarkdown: string;
   size?: IconType['iconSize'];
+  indentSize?: number;
```

Comment:
Can you pass `marginRight` as the property instead of `indentSize` and use `theme.gridUnit * 2` as its default value? It seems more clear what's the objective. It's hard to grasp that the indent will be added to the margin.